onnxruntime/core/optimizer/transpose_optimizer/transpose_optimizer.cc

```diff
@@ -1901,7 +1910,8 @@ bool OptimizeImpl(OptimizerCtx& ctx) {
   // Currently limiting the second optimization pass to layout transform mode
   // TODO: Enable this for both the modes.
   if (ctx.mode == OptimizerMode::OPTIMIZE_TRANSPOSE) {
-    return changed;
```

Comment:
>  return changed;

nit: not related to this pr,  just while reading a couple places used similar naming `changed`, `modified`, maybe would be better/clearer to have it specified/unified.